src/main/java/com/synopsys/integration/detect/workflow/bdio/CodeLocationBdioCreator.java

```diff
@@ -51,11 +69,32 @@ public CodeLocationBdioCreator(final DetectBdioWriter detectBdioWriter, final Si
             final ExternalId externalId = bdioCodeLocation.getDetectCodeLocation().getExternalId();
             final DependencyGraph dependencyGraph = bdioCodeLocation.getDetectCodeLocation().getDependencyGraph();
 
+            // Bdio 1
             final SimpleBdioDocument simpleBdioDocument = simpleBdioFactory.createSimpleBdioDocument(codeLocationName, projectNameVersion.getName(), projectNameVersion.getVersion(), externalId, dependencyGraph);
 
-            final File outputFile = new File(bdioOutput, bdioCodeLocation.getBdioName());
-            detectBdioWriter.writeBdioFile(outputFile, simpleBdioDocument);
-            uploadTargets.add(UploadTarget.createDefault(codeLocationName, outputFile));
+            final File bdio1OutputFile = new File(bdioOutput, bdioCodeLocation.getBdioName() + ".jsonld");
+            detectBdioWriter.writeBdioFile(bdio1OutputFile, simpleBdioDocument);
+            uploadTargets.add(UploadTarget.createDefault(codeLocationName, bdio1OutputFile));
+
+            // Bdio 2
+            final ProductList.Builder productListBuilder = new ProductList.Builder();
+            final String detectVersion = detectInfo.getDetectVersion();
+            final SpdxCreator detectCreator = SpdxCreator.createToolSpdxCreator("Detect", detectVersion);
+            final Product product = new Product.Builder().name(detectCreator.getIdentifier()).build();
+            productListBuilder.addProduct(product);
+
+            final BdioMetadata bdioMetadata = bdio2Factory.createBdioMetadata(codeLocationName, ZonedDateTime.now(), productListBuilder);
+            final Project bdio2Project = bdio2Factory.createProject(externalId, projectNameVersion.getName(), projectNameVersion.getVersion());
+            final Bdio2Document bdio2Document = bdio2Factory.createBdio2Document(bdioMetadata, bdio2Project, dependencyGraph);
+
+            final Bdio2Writer bdio2Writer = new Bdio2Writer();
+            final File bdio2OutputFile = new File(bdioOutput, bdioCodeLocation.getBdioName() + ".bdio");
```

Comment:
This .bdio file would be a zip file, the Blackduck endpoint for Polaris is good to consume chunks of the data and not the whole zip file.

Reply:
Can Polaris consume the zipped .bdio format?

Reply:
We are currently planning to only output the .bdio archive for now. If this is a problem for Polaris we need to reconsider our approach.